src/requests/session.ts

```diff
@@ -16,151 +25,210 @@ export class SessionClient {
     this.client = client;
   }
 
-  async find(sessionId: string): Promise<Session | null> {
-    return getSessionById(this.client, sessionId);
-  }
-
-  async findDetails(sessionId: string): Promise<DetailedSession | null> {
-    return getDetailedSessionById(this.client, sessionId);
-  }
-
   async create({
     flowId,
-    data,
-    email = "",
+    breadcrumbs,
+    email,
   }: {
-    flowId: string;
-    data: Partial<SessionData>;
+    flowId: FlowId;
+    breadcrumbs: OrderedBreadcrumbs;
     email?: string;
-  }) {
-    return createSession({ client: this.client, flowId, data, email });
+  }): Promise<SessionId> {
+    const sessionId: SessionId = uuidV4();
+    await Promise.all([
```

Comment:
It may be slightly more complex / less readable, but I think we'd be better treating these CRUD operations on `sessions` and `lowcal_sessions` as single transaction (i.e. two mutations within the same query).